lexicons/app/bsky/admin/takeModerationAction.json

```diff
@@ -0,0 +1,30 @@
+{
+  "lexicon": 1,
+  "id": "app.bsky.admin.takeModerationAction",
```

Comment:
one more bikeshed: arguably these should be in the `atproto` namespace 🤔 

Like server takedowns seem general, but as we introduce concepts of zones/labels/etc, those seem more bsky

Reply:
Sounds good.  In that case I think we just need to ensure we're happy moving forward making the method sufficiently generalized, i.e. that we can apply a takedown to any record (versus e.g. being specific to posts).

That might encourage us to write other methods in a generalized fashion, too, and that seems totally reasonable but comes with its drawbacks.  For example, if we have a method that we want to use to list all posts that have been moderated, that method would probably receive a collection name as a parameter, and return raw post records rather than the views of post records that app.bsky provides.

My inclination was to start with specialized tooling for posts and work backwards into more generalized tooling, but not at all married to that approach!

Reply:
Thinking a little more, I suppose we could put actor and record takedown procedures in com.atproto and still place moderation views in app.bsky as needed.  It would still be app.bsky's job to observe those moderation actions and apply them to its indexes, but that seems reasonable enough.

One snag still coming to mind— there could be a case where a moderation action of a record has some meaning but isn't implemented.  E.g. if we allow moderating any record in com.atproto, and you takedown something very specific like a follow or vote record, there's a good chance that app.bsky doesn't actually implement that takedown in its views, and it has no practical effect.

Reply:
This ended-up being the call 👍 the server-level moderation actions are based around repos and records via com.atproto.